app/views/katello/api/v2/subscriptions/show.json.rabl

```diff
@@ -59,10 +59,3 @@ node :activation_keys do |subscription|
     }
   end
 end
-
-node :host, :if => lambda { |sub| sub && sub.host } do |subscription|
```

Comment:
So i actually removed this because it was there in base:  https://github.com/Katello/katello/blob/master/app/views/katello/api/v2/subscriptions/base.json.rabl#L19-L24

I'm wondering if its actually used  there even now.  @thomasmckay @johnpmitsch either of you know?


As for systems, I'm wondering if this should even be there at all.  This will be really slow if there are a lot of systems, and I'd rather not re-implement it for hosts in the same broken way.  Should probably be replaced by http://projects.theforeman.org/issues/11505

 Thoughts @thomasmckay ?